pkg/component/manager_test.go

```diff
@@ -52,29 +52,32 @@ func Test_ResolvePath(t *testing.T) {
 		test.StageFile(t, fs, "deployment.yaml", "/app/components/nested/deployment.yaml")
 
 		cases := []struct {
-			name   string
-			cName  string
-			module string
-			isErr  bool
+			name              string
+			expectedComponent string
+			expectedModule    string
+			isErr             bool
 		}{
 			{
-				name:   "/",
-				module: "/",
+				name:           "/",
```

Comment:
so .. this works, but contextually being able to specify a path with / in it to ResolvePath but also that names cannot have / in them feels a bit rough to me - can we go all dots?

Reply:
I thought about all dots, but having one named "." felt weird to me, so I left it as "/".